tests/iio_readdev.c

```diff
@@ -378,6 +380,12 @@ int main(int argc, char **argv)
 		return EXIT_FAILURE;
 	}
 
+#ifdef _WIN32
+	// Deactivate the translation for the stdout. Otherwise, bytes that have
+	// the same value as line feed character (LF) will be translated to CR-LF.
+	_setmode(_fileno(stdout), _O_BINARY );
```

Comment:
nitpicks only:
1. `_setmode(_fileno(stdout), _O_BINARY );` ->   `_setmode(_fileno(stdout), _O_BINARY);`  ; remove space before last round bracket
2. generally, C-style comments are recommended; something like
```
	/*
	 * Deactivate the translation for the stdout. Otherwise, bytes that have
	 * the same value as line feed character (LF) will be translated to CR-LF.
	 */
```

Reply:
At the moment I was under the impression that I'm looking at C++ code.
Thanks!